custom_components/adaptive_lighting/const.py

```diff
@@ -64,6 +64,9 @@
 TURNING_OFF_DELAY = 5
 CONF_SEND_SPLIT_DELAY, DEFAULT_SEND_SPLIT_DELAY = "send_split_delay", 0
 
+CONF_STRICT_ADAPTING, DEFAULT_STRICT_ADAPTING = "strict_adapting", True
+CONF_ALT_DETECT_METHOD, DEFAULT_ALT_DETECT_METHOD = "alt_detect_method", False
+CONF_AUTORESET_CONTROL, DEFAULT_AUTORESET_CONTROL = "autoreset_control_time", 0
```

Comment:
Nitpick: Consider renaming to `autoreset_control_seconds` (or something similar) so the metric is clear without needing to reference the definition in string.json.